packages/logs/src/boot/preStartLogs.ts

```diff
@@ -66,6 +67,12 @@ export function createPreStartStrategy(
       }
 
       cachedConfiguration = configuration
+      // Instrumuent fetch to track network requests
+      // This is needed in case the consent is not granted and some cutsomer
+      // library (Apollo Client) is storing uninstrumented fetch to be used later
+      // The subscrption is needed so that the instrumentation process is completed
+      initFetchObservable().subscribe(noop)
```

Comment:
We talked a bit about it during standup. I'll try to sum it up here.

On one side, we all agree that `.subscribe(noop)` is a bit suprising (but not necessarily hacky). On the other side, changing this would make things more complex in `fetchObservable.ts`. 

What you are suggesting is incorrect because if `stop` is called because there is no more subscriber, then the observable is subscribed again, we wouldn't re-instrument `fetch`. In practice this probably does not happen, but for a newcomer reading this code without context this might be seen as a bug.

So we would need another way to clean the instrumentation? Or maybe don't clean the instrumentation? It doesn't feel great either.

Reply:
Well I think `.subscribe(noop)` does not convey the intention of what it does and raise more questions about the implementation for new newcomer reading this code.  
I put the `stop` in the observable only to avoid modifying too much the tests. We could have also extended the `resetFetchObservable` to reset the instrumentation. Since the removal of the instrumentation and the reset of the observable are only done for testing purposes, this approach would have been reasonable. I feel like we're choosing a confusing implementation just to avoid refactoring the tests.